assets/static/images/icon-alert-circle.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path d="m24 0a24 24 0 1 0 17 7 23.93 23.93 0 0 0 -17-7z" fill="#fff" fill-rule="evenodd"/><circle cx="24" cy="24" fill="#606060" r="22.59"/><path d="m23.89 46.59a22.59 22.59 0 1 0 -22.48-22.7 22.59 22.59 0 0 0 22.48 22.7z" fill-rule="evenodd"/><g fill="#fff"><path d="m20.39 4.42h7.22a1.81 1.81 0 0 1 1.49 2.11l-1.49 23.22a1.73 1.73 0 0 1 -1.49 1.78h-4.24a1.73 1.73 0 0 1 -1.49-1.78l-1.49-23.22c-.07-1.13.61-2.11 1.49-2.11"/><path d="m21.57 34.54h4.87a1.88 1.88 0 0 1 1.82 1.93v5.17a1.89 1.89 0 0 1 -1.82 1.94h-4.87a1.89 1.89 0 0 1 -1.83-1.94v-5.17a1.88 1.88 0 0 1 1.83-1.93" fill-rule="evenodd"/></g></svg>
```

Comment:
Do you think it's reasonable to think about this when implementing the on-ladder icons? I feel like the person implementing that will have a better ability to make these decisions because they will be working through whatever needs there are at that point.

Reply:
Good point. We'll need to keep in mind that now there are 2 places to update when setting those colors (in the SVG and in `_block_wavier.scss`) but that will be easier than trying to predict what styling we'll need in the future.